pkg/nexus/nexus/nexus-router.go

```diff
@@ -25,6 +26,7 @@ func (nexusRouter *NexusRouter) Initialize() {
 	nexusRouter.Router.Post("/scheduler/{schedulerName}/start", nexusRouter.StartScheduler)
 	nexusRouter.Router.Post("/scheduler/{schedulerName}/stop", nexusRouter.StopScheduler)
 	nexusRouter.Router.Get("/scheduler", nexusRouter.GetAllSchedulersWithStatus)
+	nexusRouter.Router.Put("/nexus-config", nexusRouter.ModifyNexusConfig)
```

Comment:
now the route is /api/nexus/nexus-config

maybe use /api/nexus/config
-> 	nexusRouter.Router.Put("/config", nexusRouter.ModifyNexusConfig)

Reply:
Guter Hinweis danke